cookbook-api/server/GraphQLSchema.ts

```diff
@@ -4,7 +4,7 @@ import { GraphQLObjectType,
     GraphQLList,
     GraphQLString } from 'graphql';
 import { resolver } from "graphql-sequelize";
-const models = require("../database-migrations/models");
```

Comment:
thanks for fixing. I also fixed as part of my changes that have already been merged, so I'm going to decline this one.